Airwallex/CardTests/AWXCardViewModelTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AWXCardViewModelTests.swift
+//  CardTests
+//
+//  Created by Tony He (CTR) on 2024/8/2.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+import UIKit
+import XCTest
+
+@testable import Card
+
+class AWXCardViewModelTests: XCTestCase {
+    func testEnableTaoEndEditing() {
+        let session = AWXOneOffSession()
+        let cardSchemes = ["visa", "mastercard", "unionpay", "jcb", "diners", "discover"].map {
+            let card = AWXCardScheme()
+            card.name = $0
+            return card
+        }
+        let viewModel = AWXCardViewModel(session: session, supportedCardSchemes: cardSchemes)
+
+        if let brands = viewModel.makeDisplayedCardBrands() as? [AWXBrandType] {
+            XCTAssertEqual(brands.count, 6)
+            XCTAssertEqual(brands[0], AWXBrandType.visa)
+            XCTAssertEqual(brands[1], AWXBrandType.mastercard)
+            XCTAssertEqual(brands[2], AWXBrandType.discover)
+            XCTAssertEqual(brands[3], AWXBrandType.JCB)
+            XCTAssertEqual(brands[4], AWXBrandType.dinersClub)
+            XCTAssertEqual(brands[5], AWXBrandType.unionPay)
+        }
```

Comment:
Maybe add
`else {
   XCTFail
}` just in case tests are bypassed?

Reply:
Change the function return value into [Int] to avoid unwrapping here.